contracts/partials/strategy/helpers.ligo

```diff
@@ -85,11 +109,13 @@ function operate_with_strategy(
           };
           strategy.configuration[token_id] := config;
         };
+        require(List.size(rebalance_params) > 0n, Errors.Strategy.nothing_to_rebalance);
```

Comment:
We shouldn't fail, but skip sending operations. Otherwise, when we will use the method in swap/invest/divest, we will fail every time when the rebalance isn't needed.

Reply:
Ok, got it) Used it to debug some cases and will remove it